### PR TITLE
Fixing issue 544(wrong instructions on beta) and 488 (removing now from the footer)

### PR DESCRIPTION
**Pull request by bogomil**
There is my proposal for fixing the issue #544 until the new website is prepared!

### CODE REVIEW

**Comment by bogomil** [comment]
@MelissaAutumn  and @Sancus for a review!

File: website/download/beta/index.html [hunk 1]
```diff
@@ -144,7 +144,7 @@ <h2 class="header-section pl-4 pr-4 mb-0 text-center">{{ _('Resources and Assist
...
         <li>{{ _('If you find a problem, try to <a href="https://support.mozilla.org/kb/troubleshoot-mode-thunderbird">reproduce the problem in Troubleshoot Mode</a>, then file a bug report for <a href="https://bugzilla.mozilla.org/enter_bug.cgi?product=Thunderbird">Thunderbird</a> or <a href="https://bugzilla.mozilla.org/enter_bug.cgi?product=Calendar">Calendar</a>.') }}</li>
         <li>{{ _('<b>≡ > Tools > Export</b> can be used to backup your profile, or to get to your profile directory quickly.') }}</li>
-        <li>{{ _('<b>≡ > Go > Mail Start Page (aka Beta Start Page)</b> is a handy reference which documents several of the above items and more.') }}</li>
+        <li>{{ _('<b> Invoke the Mail Start Page (aka Beta Start Page)</b> shortcut listed <a href="https://support.mozilla.org/kb/keyboard-shortcuts-thunderbird#w_moving-around-thunderbird">here</a> to open a handy reference that documents several of the above items and more.') }}</li>
```
Comment (MelissaAutumn) [inline website/download/beta/index.html:147]:
```suggestion
        <li>{{ _('<b> Invoke the Mail Start Page (aka Beta Start Page)</b> with the shortcut listed <a href="https://support.mozilla.org/kb/keyboard-shortcuts-thunderbird#w_moving-around-thunderbird">here</a> to open a handy reference that documents several of the above items and more.') }}</li>
```

**Review by MelissaAutumn — COMMENTED** [review]
Just one tiny change. Looks good to me otherwise, I'll merge it Monday. Thanks!

**Comment by bogomil** [comment]
Thanks!

**Pull request merged**